test/chainsaw/traces/verify-traces.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
I am wondering if we can simplify this bash script in favour of a more chainsaw flavoured check, something like in here https://kyverno.github.io/chainsaw/latest/examples/test-output/?h=check#checking-output-is-exactly

Reply:
this is valid, I will try it out!

Reply:
it might be complicated, as in the script we are trying to reach 13 different substrings in the output, so if I understand it correctly, we need to execute kubectl logs 13 times and match the output, it's more sufficient in this case to do it via script

Reply:
ah so they are sparse around the output? I guessed you can have multiple checks for each so it could be a list of checks within the same single log command

Reply:
it's working like a regex match also with retry logic which perfectly makes sense here due to that information might not be available right away in the logs. I think the check does not have the needed complexity here. It's awesome for simple checks, but for this case I would go with the script